eve/main.yml

```
@@ -125,6 +126,29 @@ stages:
         urls:
           - "*.iso"
 
+  build-base-iso:
+    worker:
+      type: kube_pod
+      path: eve/workers/build-iso/pod.yaml
+      images:
+        build-iso: eve/workers/build-iso
```

I don't know if you want to use that weird yaml pointer thing (forgot the technical name of it), but in case you do. I suggest that you set on line 107

```yaml
    worker: &build-iso-worker
```

so that you can do the following:
```suggestion
    worker: *build-iso-worker
```

---

oh let me try these